aspnetcore/security/data-protection/includes/introduction7.md

```diff
@@ -1,5 +1,8 @@
 :::moniker range="< aspnetcore-8.0"
 
+> [!WARNING]
+> Data configuration keys in this version of .NET do ***NOT*** maintain data consistency by default in web farms and [Azure Container Apps](/azure/container-apps/overview). See the the [.NET 8 version or later of this document](/aspnet/core/security/data-protection/introduction?view=aspnetcore-800.0&preserve-view=true) for information on how to maintain data consistency in web farms and Azure Container Apps.
```

Comment:
It feels strange to warn that 7.0, which is out of support, will retain the behavior it has always had.  It seems like such a warning would chiefly benefit people writing new apps and hopefully those people aren't using 7.0.

Reply:
If we were to keep this warning, e.g. in the 6.0 docs, I think I might say something like "New functionality was introduced in AspNetCore 8.0 to automatically keep Data Protection keys synchronized across instances in Azure Container Apps.  That functionality is not present in 6.0, but information about how to configure it manually can be found [here](https://learn.microsoft.com/aspnet/core/host-and-deploy/scaling-aspnet-apps/scaling-aspnet-apps?view=aspnetcore-6.0&tabs=login-azure-cli)."

Having said that, putting it before the problem statement in an overview doc feels very strange to me.  If there's a place where the docs say that something will happen automatically (it may mention defaulting to file storage), I would think _that_ would be the place for a note about ACA (and maybe a warning if it won't work in a particular version of aspnetcore).